SimPEG/PF/Magnetics.py

```diff
@@ -79,6 +79,11 @@ def G(self):
 
         return self._G
 
+    @property
+    def modelMap(self):
```

Comment:
I think we might want to make this private (eg `_model_map`) as we want to promote users working with the physical properties, but internally, it can be handy for inheriting from the linear problem